tests/util/test_async_helpers.py

```diff
@@ -60,11 +60,9 @@ def check_called_first(res: int) -> int:
         observer1.addBoth(check_called_first)
 
         # store the results
-        results: List[Optional[ObservableDeferred[int]]] = [None, None]
+        results: List[Optional[int]] = [None, None]
 
-        def check_val(
-            res: ObservableDeferred[int], idx: int
-        ) -> ObservableDeferred[int]:
+        def check_val(res: int, idx: int) -> int:
```

Comment:
I checked this was correct by running the test with `print(type(res))` inserted.